permissions/plugin-configuration-as-code-integrations.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "integrations"
+github: "jenkinsci/configuration-as-code-plugin"
+paths:
+- "io/jenkins/configuration-as-code/integrations"
```

Comment:
Now I'm wondering if the permission schema is recursive, and if so putting `io/jenkins/configuration-as-code` only wouldn't hence make things possibly simpler.

Reply:
It is not.